DataConverters3D/Object3D/Object3D.cs

```diff
@@ -260,13 +261,20 @@ public PrintOutputTypes OutputType
 		public Matrix4X4 Matrix { get; set; } = Matrix4X4.Identity;
 
 		[JsonIgnore]
-		public Mesh Mesh { get; set; }
-
-		public void SetAndInvalidateMesh(Mesh mesh)
+		private Mesh _mesh;
+		public Mesh Mesh
 		{
-			this.Mesh = mesh;
-			this.MeshPath = null;
-			this.traceData = null;
+			get => _mesh;
+			set
+			{
+				if (_mesh != value)
+				{
+					_mesh = value;
+					traceData = null;
+					this.MeshPath = null;
+					this.OnInvalidate();
```

Comment:
IObject3D didn't have Invalidate last time this code was touched. Now that it does, I believe we want to fire the Invalidate event when the mesh changes and this extra call to OnInvalidate now does